Create 2022-04-26-email-verification.md (#1384)
fishtown-analytics/docs.getdbt.com: website/blog/2022-04-26-email-verification.md

---
title: "Email verification"
description: "dbt Cloud users created via SAML JIT will now confirm identity via email to activate their account."
slug: 2022-04-26-email-verification

tags: [release notes]
hide_table_of_contents: false

date: 2022-04-26

> is_featured: true

Author: ```suggestion
is_featured: false
```

---

> dbt Cloud users created via SAML JIT will now confirm identity via email to activate their account. This enhances the security of user creation by ensuring the user still has access to the same email address they use to login via SAML. 

Author: ```suggestion
To enhance the security of user creation, dbt Cloud users created using SAML Just-in-Time (JIT) will now confirm identity via email to activate their account. Using email to confirm identity ensures the user still has access to the same email address they use to login via SAML. 
```
